### PR TITLE
Add vehicle-attributes-recognition-barrier-0042 model

### PR DESCRIPTION
**Pull request by LeonidBeynenson**

### CODE REVIEW

**Comment by LeonidBeynenson** [comment]
2 @vladimir-dudnik :    
Valdimir, please, take a look

**Comment by LeonidBeynenson** [comment]
2 @vladimir-dudnik :    
Vladimir, I'm sorry, but I see a lot of similar errors in documentation in the files which I did not change.   
Example:

```
22:37:42  [Error] Found broken links in intel/pedestrian-and-vehicle-detector-adas-0001/description/pedestrian-and-vehicle-detector-adas-0001.md
```

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+| Metric                | Value                                        |
+|-----------------------|----------------------------------------------|
+| Car pose              | Front facing cars                            |
+| Occlusion coverage    | <50%                                         |
+| Min object width      | 72 pixels                                    |
+| Supported colors      | White, gray, yellow, red, green, blue, black |
+| Supported types       | Car, van, truck, bus                         |
+| GFlops                | 0.462                                        |
+| MParams               | 11.177                                       |
+| Source framework      | Pytorch\*                                    |
```
Comment (eizamaliev) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:22]:
```suggestion
| Source framework      | PyTorch\*                                    |
```

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+## Accuracy (per class)
```
Comment (eizamaliev) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:24]:
```suggestion
## Accuracy
```

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+| Type  | Accuracy |
+|:-----:|:--------:|
+| car   | 97.96%   |
+| van   | 86.08%   |
+| truck | 97.47%   |
+| bus   | 42.49%   |
+
+**Type average accuracy: 81.00%**
+
+## Performance (FPS)
```
Comment (eizamaliev) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:51]:
```suggestion
## Performance
```

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+## Inputs
+
+1.	name: `input` , shape: [1x3x72x72] - An input image in following format
```
Comment (eizamaliev) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:55]:
```suggestion
1.	Name: `input` , shape: [1x3x72x72] - an input image in following format
```

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+[1xCxHxW], where:
+
+    - C - number of channels
+    - H - image height
+    - W - image width
```
Comment (eizamaliev) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:60]: There is no need in such indentation.

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+Expected color order: BGR.
+
+## Outputs
+
+1.	name: "color", shape: [1, 7, 1, 1] - Softmax output across seven color classes
```
Comment (eizamaliev) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:66]:
```suggestion
1.	Name: `color`, shape: [1, 7, 1, 1] - probabilities across seven color classes
```
*Here and below*

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md (hunk 2 above)
Comment (eizamaliev) [reply models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:24]: This is common header for that section

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+    [white, gray, yellow, red, green, blue, black]
```
Comment (eizamaliev) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:67]: @LeonidBeynenson what do you think about emphasizing class names in some way?

**Comment by LeonidBeynenson** [comment]
Jenkins please retry a build

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+| truck | 97.47%   |
+| bus   | 42.49%   |
+
+**Type average accuracy: 81.00%**
+
+## Performance
+
+## Inputs
+
+1.	Name: `input` , shape: [1x3x72x72] - an input image in following format
```
Comment (eizamaliev) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:55]:
```suggestion
Name: `input` , shape: [1x3x72x72] - an input image in following format
```

File: tools/accuracy_checker/configs/vehicle-attributes-recognition-barrier-0042.yml [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+          - FP16-INT8
+        model:   intel/vehicle-attributes-recognition-barrier-0042/FP16-INT8/vehicle-attributes-recognition-barrier-0042.xml
+        weights: intel/vehicle-attributes-recognition-barrier-0042/FP16-INT8/vehicle-attributes-recognition-barrier-0042.bin
+        adapter:
+          type: vehicle_attributes
+          color_out: color
+          type_out: type
+
+    datasets:
+      - name: vehicle_attributes
```
Comment (eaidova) [inline tools/accuracy_checker/configs/vehicle-attributes-recognition-barrier-0042.yml:36]: proper dataset configuration should be discussed offline

Comment (eaidova) [inline tools/accuracy_checker/configs/vehicle-attributes-recognition-barrier-0042.yml:36]:
```suggestion
      - name: vehicle_attributes_0042
```

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+- H - image height
+- W - image width
+
+Expected color order: BGR.
+
+## Outputs
+
+1.	Name: `color`, shape: [1, 7, 1, 1] - probabilities across seven color classes
+    [white, gray, yellow, red, green, blue, black]
+2.	name: "type", shape: [1, 4, 1, 1] - Softmax output across four type classes
```
Comment (eaidova) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:67]: for alignment with `color` output
```suggestion
2.	name: `type`, shape: [1, 4, 1, 1] - Softmax output across four type classes
```

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md (hunk 7 above)
Comment (eaidova) [reply models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:66]: for example:
```suggestion
    [`white`, `gray`, `yellow`, `red`, `green`, `blue`, `black`]
```
or
```suggestion
    - white,
    - gray
    - yellow
    - red,
    - green
    - blue,
    - black
```

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+    [car, van, truck, bus]
```
Comment (LeonidBeynenson) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:68]:
```suggestion
    [`car`, `van`, `truck`, `bus`]
```

**Comment by LeonidBeynenson** [comment]
It turned out that several my github comments to this PR were in pending state. 🙁
(and this my comment was accidentally gone to another PR, sorry)
Now I sent them and applied all the suggested changes

2 @eizamaliev , @eaidova :
Could you check if everything is OK now? can this PR be merged?

**Comment by LeonidBeynenson** [comment]
NB: the documentation build reports errors in the files that were not changed by me!

**Comment by eaidova** [comment]
> 2 @eizamaliev , @eaidova :
> Could you check if everything is OK now? can this PR be merged?

I approved, let's wait @eizamaliev  decision




> NB: the documentation build reports errors in the files that were not changed by me!

Yes, looks like one of outdated reid  model was removed from docs

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+- H - image height
+- W - image width
+
+Expected color order: BGR.
+
+## Outputs
+
+1.	Name: `color`, shape: [1, 7, 1, 1] - probabilities across seven color classes
+    [`white`, `gray`, `yellow`, `red`, `green`, `blue`, `black`]
+2.	name: `type`, shape: [1, 4, 1, 1] - Softmax output across four type classes
```
Comment (LeonidBeynenson) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:67]:
```suggestion
2.	name: `type`, shape: [1, 4] - probabilities across four type classes
```

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+[1xCxHxW], where:
+- C - number of channels
+- H - image height
+- W - image width
+
+Expected color order: BGR.
+
+## Outputs
+
+1.	Name: `color`, shape: [1, 7, 1, 1] - probabilities across seven color classes
```
Comment (LeonidBeynenson) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:65]:
```suggestion
1.	Name: `color`, shape: [1, 7] - probabilities across seven color classes
```

File: models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+- H - image height
+- W - image width
+
+Expected color order: BGR.
+
+## Outputs
+
+1.	Name: `color`, shape: [1, 7] - probabilities across seven color classes
+    [`white`, `gray`, `yellow`, `red`, `green`, `blue`, `black`]
+2.	name: `type`, shape: [1, 4] - probabilities across four type classes
```
Comment (eizamaliev) [inline models/intel/vehicle-attributes-recognition-barrier-0042/description/vehicle-attributes-recognition-barrier-0042.md:67]:
```suggestion
2.	Name: `type`, shape: [1, 4] - probabilities across four type classes
```

**Pull request merged**